src/Sarif/Writers/CacheByFileHashLogger.cs

```diff
@@ -15,17 +15,16 @@ namespace Microsoft.CodeAnalysis.Sarif.Writers
     /// of simply repeating the analysis. This can result in significant performance gains, when that analysis
     /// is expensive (such as in the case of a binary analysis that retrieves and crawls binary PDBs).
     /// </summary>
-    public class CacheByFileHashLogger : IAnalysisLogger
+    public class CacheByFileHashLogger : BaseLogger, IAnalysisLogger
     {
         private bool cacheLoggingData;
         private string currentFileHash;
 
         public Dictionary<string, List<Notification>> HashToNotificationsMap { get; private set; }
         public Dictionary<string, List<Tuple<ReportingDescriptor, Result>>> HashToResultsMap { get; private set; }
 
-        public CacheByFileHashLogger(bool verbose)
+        public CacheByFileHashLogger(IEnumerable<FailureLevel> level, IEnumerable<ResultKind> kind) : base(level, kind)
```

Comment:
>level [](start = 63, length = 5)

s/be plural names

Reply:
Fixed.

---
In reply to: [559747791](https://github.com/microsoft/sarif-sdk/pull/2241#discussion_r559747791) [](ancestors = 559747791)